pipeline: processors: conditional processing: general cleanup (#1887)
fluent/fluent-bit-docs: pipeline/processors/conditional-processing.md

# Conditional processing

Conditional processing lets you selectively apply [processors](README.md) to logs based on the value of fields within those logs. This feature lets you create processing pipelines that only process records that meet certain criteria, and ignore the rest.

Conditional processing is available in Fluent Bit version 4.0 and greater.

## Configuration

You can turn a standard processor into a conditional processor by adding a `condition` block to the processor's YAML configuration settings.

{% hint style="info" %}

> - Conditional processing is only available for [YAML configuration files](../administration/configuring-fluent-bit/yaml/README.md), not [classic configuration files](../administration/configuring-fluent-bit/classic-mode/README.md).
> - Conditional processing isn't supported if you're using a [filter as a processor](../pipeline/processors/filters).

Author: ```suggestion
- Only [YAML configuration files](../../administration/configuring-fluent-bit/yaml/README.md) support processors, including conditional processors.
- Conditional processing isn't supported if you're using a [filter as a processor](../pipeline/processors/filters).
```
